flashlight/pkg/vision/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.10)
+
+add_library(fl_pkg_vision "")
+
+target_link_libraries(fl_pkg_vision PUBLIC flashlight)
+
+target_include_directories(fl_pkg_vision PUBLIC)
```

Comment:
remove? everything should work via linking to the `flashlight` target

Reply:
Yep, good catch. I don't think `target_include_directories(fl_pkg_vision PUBLIC)` is actually doing anything.